.github/workflows/setup_python_conda.yml

```diff
@@ -0,0 +1,35 @@
+name: set up python environment
+on: push
+
+jobs:
+  setEnv:
+    name: set up python conda environment
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: s-weigand/setup-conda@v1
+        with:
+          activate-conda: true
+      - name: install Python dependencies
+        run: |
+          conda create -n manyFEMS
+          conda install numpy
+          conda install xlrd==1.2.0
+
+      - name: test python environment and 'hello world' CI
+        run: |
+          which python
+          echo hellow world
+          python GenerateRiverFlows/hellow_world.py
+
```

Comment:
We can probably get rid of these lines (and the hellow_world.py file) now we have the GenerateRiverFlows.py script working in CI.

Reply:
accepted and corrected.